bin/grakn

```diff
@@ -69,15 +69,14 @@ elif [[ "$1" = "server" ]] || [[ "$1" = "version" ]]; then
     else
         echo "Grakn Core Server is not included in this Grakn distribution."
         echo "You may want to install Grakn Core Server or Grakn Core (all)."
+        exit 1
     fi
 else
     echo "Invalid argument: $1. Possible commands are: "
     echo "  Server:          grakn server [--help]"
     echo "  Console:         grakn console [--help]"
+    exit 1
 fi
 
-exit_code=$?
-
-popd > /dev/null
```

Comment:
Can we keep the `popd`? Since we're doing a `pushd` at the top I think it's nice to keep it (though I know it's not neccessary for correctness since it's located at the end of the file)

Reply:
@lolski `popd` effectively is a no-op since our script runs in a subshell. It has no effect on the calling shell

Reply:
Yeah, I just find it weird that we're doing something and not put it back. It's like `open()`-ing a file but not calling `close()` afterwards.

If you prefer to remove it, can you remove the on in KGMS as well to maintain consistency?

Reply:
@lolski 
>If you prefer to remove it, can you remove the on in KGMS as well to maintain consistency?

I'll do the same in PR to KGMS

Reply:
As discussed, let's also change `pushd` to plain old `cd` in both this PR and the one to KGMS.

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/5390/commits/52189c5ddae1958cee0ca364b8e73a45d9c52f32